model/map-jpa/src/main/resources/META-INF/clients/jpa-clients-changelog-1.xml

```diff
@@ -29,6 +29,7 @@ limitations under the License.
             <column name="id" type="UUID">
                 <constraints primaryKey="true" nullable="false"/>
             </column>
+            <column name="version" type="INTEGER"/>
```

Comment:
Should this column be not null with default value 0?

Reply:
As this is the initial setup of the table with no data in it, the default value would be optional. 
+1 for "not null". 

If we want consistency on this for version columns, there should be a unit test for that (but that should be a separate GitHub issue).